### DISCUSSION TITLE
Opt out of new dashboard

### DISCUSSION THREAD

**Discussion by mrjones2014** [community/community#53712] [Other Feature Feedback, Questions, & Ideas]
### Dashboard

Product Feedback

### Body

New dashboard is infinitely less useful and more annoying. Please give me the old dashboard back.

I don't want to see a bunch of random crap I didn't subscribe to. This used to be contained in a "For You" tab or whatever which I blocked with uBlock Origin. Now its in my main feed.

Please revert it or give me an option to opt out of it. GitHub isn't Facebook. I only want to see things I've subscribed to on my dashboard.

**Comment by mrjones2014** [comment]
Actually, my uBlock Origin filter just got messed up because you made "For You" the default.

**Please, please, please** give us an option to set the default back to "following".

**Reply by mrjones2014** [reply]
I effectively cannot block the "For You" tab anymore because if I log in on a new device, it will default to "For You", but I've blocked that HTML, so I cannot turn it back to "Following" without disabling uBlock Origin.

Allow us to set a persistent default tab on our account, or opt out of the "For You" tab entirely. I have never and will never find value or click into the "For You" tab.

**Comment by jkirkpatrick** [comment]
The new dashboard just sucks. I need more useful / actionable information on the dash - not another useless social network feed.

My migration to gitlab is definitely coming in the next 3 months.

**Reply by mrjones2014** [reply]
"Some random user I don't follow contributed to some random repo I don't use and am not interested in" -- thanks for this highly relevant information... bruh moment.

**Comment by mrjones2014** [comment]
For context, there is a ton of user support for opting out completely from the algorithmic feed. I've already brought this up before, found a workable solution for myself using uBlock Origin, now you've went ahead and broken that solution for me.

https://github.com/orgs/community/discussions/13154

**Comment by gozeloglu** [comment]
Definitely, it should be optional. I don't want to see **For you** tab as default. I am not interested in it. Seeing the latest activities in order is more important for me. Please, revert back to the previous version or add an option setting.

**Comment by xaqrox** [comment]
Github is not (primarily) a social network, it's a tool. The very idea of a For You tab is actively hostile towards the productivity of users of the site. And couldn't be named more inaccurately - the things I want "For Me" are the updates from users and repositories I've followed. Haphazard engagement with algorithmically selected trending repos or users is "For *You*, Github/Microsoft". Knock it off.

**Comment by kmk3** [comment]
[uBlock Origin](https://github.com/gorhill/uBlock) filter to hide it:

```
! 2023-04-26 https://github.com
! "For you" tab and feed
github.com###feed-next
github.com##[aria-labelledby="feed-next"]
```

Edit: To clarify, I also think that the algorithmic feed should be opt-in.

**Reply by mrjones2014** [reply]
Yeah, I've reapplied basically the same thing. It's still annoying that it is the default though, so if I log in from a new browser or machine, I basically see nothing by default. There should be an option to set the default on a persistent account level.

**Comment by Secret-chest** [comment]
If you go to the Following tab, it will remember that and become the default.

**Reply by mrjones2014** [reply]
Not on a new device or if you clear cookies. Not a solution.

**Reply by Secret-chest** [reply]
> Not on a new device or if you clear cookies. Not a solution. 

I know, but how many devices/browsers do you use? It takes one click. And why'd you clear the GitHub cookies?

**Comment by stumbles** [comment]
Microsoft, get yer sleazy fingers out of github. You're recommending people I've never directly dealt with, I cannot disable "the latest changes" from those I have zero clue who they are, I don't need to see those I do follow every time they fart. You're making this site just like your garbage OS and it's pissing me off.

**Comment by philiprenich** [comment]
"For you" should not be an opt out feature, it should be an opt-in feature. Or just not be there at all, I'll take that too.
